src/pages/log-management/opensearch/opensearch-dashboards/api.mdx

```diff
@@ -0,0 +1,84 @@
+through the Logit.io Dashboard. This can improve your user experience and allow proper integration between your company, your applications 
+and the Logit.io service.
+
+You can, if you choose to, connect your application directly to Logit.io to do a range of things such as load dashboards, visualisations, 
+index patterns and mappings etc. or simply load them straight from your computer. 
+
+This article explains how to call REST API Endpoints to access Opensearch Dashboards using our Logit.io proxy and connect to it using Basic 
+Authentication.
+
+<Callout type="info">
+    Basic Authentication is a simple authentication method in HTTP(S) where the client includes a username and password with each request. 
+    The username and password is combined with a colon in-between as a string. You don't need to worry about how to create a basic authentication 
+    header as we have done it for you. 
+</Callout>
+
+Each request to a REST API Endpoint must include all of the necessary information for Opensearch Dashboards to fulfill the request. If all the necessary 
+information is provided the API requests return JSON output.
+
+Use the following operations to interact with the Opensearch Dashboards via the Logit.io Proxy.
+
+* __GET__ - Fetches the information.
+* __POST__ - Adds new information.
+* __PUT__ - Updates the existing information.
+* __PATCH__ - Applies partial modifications to the existing information.
+* __DELETE__ - Removes the information.
+
+All the details that you require in order to connect to the Logit.io proxy are found on the Profile Settings page of the Logit.io Dashboard. 
+The Profile Settings page has a section called `Proxy Settings` where you will find everything that you need.
+
+There are two ways that you can call the REST API using the Logit.io proxy. The call to our proxy needs to be in either of the following formats:
+
+__Basic Authentication header__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-H "Authorization: Basic {basic auth header}" {endpoint to call}
+```
+
+__Proxy Username and Password__:
+
+```bash
+curl -XGET "osd-xsrf:true" -H "Content-Type: application/json" 
+-u "{proxy username}:{proxy password}" {endpoint to call}
+```
+
+Below are two curl request examples that show how to connect to the Logit.io Proxy. The examples use the Stack Id of a Stack that you have access to. 
+If you need to access another of your stacks then you can simply find the Stack that you require, retrieve the stack Id from that stack and then 
+replace it. 
+
+<ProxyAuthPlaceholderWarning stackType="logs" />
+
+### Example of a curl request to access your Logit.io Opensearch Dashboards instance
+
+This first method uses the __Basic Auth Header__:
+
+```bash copy /@proxyAuthSetting.basicAuthHeader/ /@logs_id/
+curl -XGET -H "osd-xsrf:true" -H "Content-Type: application/json" -H ^
+"Authorization: Basic ^
+@proxyAuthSetting.basicAuthHeader" ^
+https://kibana.logit.io/s/@logs_id/api/status
```

Comment:
```suggestion
```bash copy /@proxyAuthSetting.basicAuthHeader/ /@logs_id/
curl -XGET -H "osd-xsrf:true" -H "Content-Type: application/json" ^
-H "Authorization: Basic @proxyAuthSetting.basicAuthHeader" ^
https://kibana.logit.io/s/@logs_id/api/status
```

Reply:
I've done it like this because when the auth header is actually filled in from the token it takes up the entire screen real estate. This formatting makes it fit (just about).

Reply:
I suggested the change because it didn't work when I tried it with the original layout. Sorry, I should have been clear that it wasn't a cosmetic change but a minor fix to make the curl command work. The change just moved the ^ to before the -H, which it seemed to be OK with.

Reply:
Ah, OK thanks @pwatson-logit . I'll make the change